index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Abhaya+Libre:wght@400;500;600;700;800&family=Exo+2:ital,wght@0,100..900;1,100..900&family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <div class="login">
+            <a href="/"><img src="/misson_1/img/sm.png" alt="logo"></a>
```

Comment:
💊 제안  
img 태그의 alt 속성은 alternative 라는 의미로, 이미지 파일을 다운로드하는 것에 실패해서 이미지 파일을 보여줄 수 없을 때  
어떤 이미지인지 파악할 수 있게 대신 제공되거나, 스크린리더로 읽혀지는 문자를 의미합니다.  
따라서 **가능한 해당 이미지를 잘 표현할 수 있는 문자열을 넣어주는 것이 좋습니다.**  
광고 배너라면 광고 배너이미지를 통해 전달하고 싶은 문구들이 alt에 들어갈 수 있습니다.

만약 해당 이미지가 내용이 없고 장식적인 이미지라면, img 태그가 아닌 css background-image 속성을 통해 해당 요소를 처리하는 것을 고려해보셔도 좋습니다.

```suggestion
            <a href="/"><img src="/misson_1/img/sm.png" alt="판다마켓"></a>
```